scripts/check_frontend_coverage.py

```diff
@@ -216,28 +216,28 @@ def __init__(self, stanza):
             Exception: Total lines number is not found.
             Exception: Covered lines number is not found.
         """
-        file_path, total_lines, covered_lines = [
+        relevant_lines = [
```

Comment:
We should avoid using x[1], x[2], etc. for stuff where the elements in the list don't represent the same thing.

Instead, you can do either of the following:
- Use three separate named variables: for the first, find the line corresponding to the filepath; for the second, find the line corresponding to total_lines, etc. Or you can even apply the regexes directly to the stanza to extract the relevant data, without needing to store the lines as an interim step.
- Put the original statement (which tries to parse the return value into a tuple) in a try/except and if you get a "cannot fit into tuple" error then handle that accordingly by raising an exception.

Reply:
Done.